src/Netflix/Netflix.ts

```diff
@@ -513,11 +510,7 @@ const handleWatchAnswer = async (command: WatchCommand) => {
   }
 
   // Filter out videos which title are empty string
-  const filteredVideos = videos.filter(
-    video => !!stripNonAlphaNumericAndTrim(video.title)
-  );
-
-  const strippedQuery = stripNonAlphaNumericAndTrim(sub.query);
+  const filteredVideos = videos.filter(video => !!video.title.trim());
```

Comment:
fuzzy high score does the trimming too. You shouldn't need to do any preprocessing of the string yourself.

Reply:
Notice that this is not `.map`. This does not change title by trimming, this removes video data which title is empty.
This relates more to data cleanup rather than data processing itself (the `fuzzyHighScore`)